x-pack/plugins/security/server/config.test.ts

```diff
@@ -0,0 +1,432 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('crypto', () => ({ randomBytes: jest.fn() }));
+
+import { first } from 'rxjs/operators';
+import { loggingServiceMock, coreMock } from '../../../../src/core/server/mocks';
+import { createConfig$, ConfigSchema } from './config';
+
+describe('config schema', () => {
+  it('generates proper defaults', () => {
+    expect(ConfigSchema.validate({})).toMatchInlineSnapshot(`
+                                                                                                            Object {
```

Comment:
__note:__ diff#9: have no idea why eslint formats inline snapshots in 7.x in a such weird way :disappointed: But it what it does automatically :man_shrugging: